CalendarApp/NUnitTestProject/BookingServiceTests.DisplayTest.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Google.Apis.Calendar.v3.Data;
+using NUnit.Framework;
+using BookingRoom.Helpers;
+using Google.Apis.Calendar.v3;
+
+namespace NUnitTestProject
+{
+    [TestFixture]
+    public partial class BookingServiceTests : BaseIntegrationTest
+    {
+        [Test]
+        public void BookingService_DisplayEvent_WrongCalendarIdThrowException()
+        {
+            var calendarId = string.Empty;//error
+
+            Assert.Throws(typeof(Google.GoogleApiException),
+               () => { _meetingBooking.GetEvents(calendarId); });
+        }
+
+
+        [Test]
+        public void BookingService_DisplayEvent_DisplaysEventCorrectly()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var actualEvents = _calendarService.Events.List(calendarId).Execute().Items;
+            var expectedEvents = _meetingBooking.GetEvents(calendarId);
+
+            CollectionAssert.AllItemsAreNotNull(expectedEvents);
+            CollectionAssert.AllItemsAreInstancesOfType(expectedEvents, typeof(Event));
+
+            for (var index = 0; index < actualEvents.Count; index++)
+            {
+                Assert.AreEqual(expectedEvents[index].Id, actualEvents[index].Id);
```

Comment:
To avoid triple array lookup for `expectedEvents[index]` and `actualEvents[index]` temporary variables can be used.